internal/decryption/kmip/client.go

```diff
@@ -306,14 +300,8 @@ func (kc *Client) GetSymmetricKey(keyID string) ([]byte, error) {
 	if err != nil {
 		return nil, fmt.Errorf("%w: %s", ErrKMIPDecodeFailure, err)
 	}
-
-	var keyValue KeyValue
-	err = decoder.DecodeValue(&keyValue, response.SymmetricKey.KeyBlock.KeyValue.(ttlv.TTLV))
-	if err != nil {
-		return nil, fmt.Errorf("%w: %s", ErrKMIPDecodeKeyBlockFailure, err)
-	}
-
-	return keyValue.KeyMaterial, nil
+	keyValue := response.SymmetricKey.KeyBlock.KeyValue
+	return keyValue.KeyMaterial.([]byte), nil
```

Comment:
was this change needed or recommended? 
I guess with the KMIP test disabled at the moment, we won't be able to test this part, but when I'll fix that test (actually the test config) I'll take this change into account (test with old / new version of the code).

Reply:
yes because of https://github.com/ThalesGroup/kmip-go/pull/19